Content.Shared/Movement/Systems/SharedMoverController.Input.cs

```diff
@@ -373,15 +373,17 @@ private void HandleRunChange(EntityUid uid, ushort subTick, bool walking)
             SetSprinting((uid, moverComp), subTick, walking);
         }
 
-        public (Vector2 Walking, Vector2 Sprinting) GetVelocityInput(InputMoverComponent mover)
+        // ES START
+        // forceWalk
```

Comment:
this should probably be more descriptive of the changes being done, think of it as an explainer for whoever has to fix it when a merge conflict happens.